terraform/azure/infra/main.tf

```diff
@@ -276,7 +316,7 @@ resource "azurerm_kubernetes_cluster" "wandb" {
   default_node_pool {
     name               = "default"
     node_count         = 2
-    vm_size            = "Standard_D4s_v5"
+    vm_size            = "Standard_D4s_v3"
```

Comment:
What's the reason for reverting?

Reply:
@venky-wandb had changed this from v3 to v5 in a separate PR by accident.  Changing this causes the entire nodepool to be torn down which causes downtime.  I couldn't find any clear benefit to potentially cause downtime so I set it back to what it was when we originally released this terraform.